pdc_client/runner.py

```diff
@@ -148,7 +148,7 @@ def setup(self):
         self.parser.add_argument('--json', action='store_true',
                                  help='display output as JSON')
         self.parser.add_argument('--page-size', dest='page_size', type=int,
-                                 help='change page size in response')
+                                 help='change page size in response, a -1 means that get all pages of data in one request')
```

Comment:
`a -1 ...` -> `-1 ...` (remove `a`)

Reply:
No sure if the sentence is okay, leave it to Chuang for having a look. @simozhan

Reply:
I agree with @hluk, 'a' is unnecessary :)

Reply:
@hluk and @simozhan changed :)